README.md

```diff
@@ -14,23 +14,27 @@ gem 'fulfil-io', require: 'fulfil'
 
 And then execute:
 
-    $ bundle install
+```shell
+  $ bundle install
+```
 
 Or install it yourself as:
 
-    $ gem install fulfil-io
+```shell
+  $ gem install fulfil-io
+```
 
 ## Usage
 
 Environment variables:
 
-- FULFIL_SUBDOMAIN - required to be set.
-- FULFIL_TOKEN - required for oauth bearer authentication
-- FULFIL_API_KEY - required for authentication via the X-API-KEY request header
+- **FULFIL_SUBDOMAIN:** - always required to use the gem.
+- **FULFIL_OAUTH_TOKEN:** required for oauth bearer authentication
+- **FULFIL_API_KEY:** required for authentication via the `X-API-KEY` request header
 
-**Note:** When FULFIL_TOKEN is present, the FULFIL_API_KEY will be ignored. So,
+> **Note:** When `FULFIL_OAUTH_TOKEN` is present, the `FULFIL_API_KEY` will be ignored. So,
 if oauth doesn't work, returning an Unauthorized error, to use the
-FULFIL_API_KEY, the FULFIL_TOKEN shouldn't be specified.
+`FULFIL_API_KEY`, the `FULFIL_OAUTH_TOKEN` shouldn't be specified.
```

Comment:
This is super helpful.

Reply:
I was also thinking about renaming the `FULFIL_SUBDOMAIN` to `FULFIL_MERCHANT_ID` as that's how Fulfil calls it. For now, I kept it the same, but we might want to consider updating that variable as well.